receiver/splunkhecreceiver/receiver.go

```diff
@@ -327,6 +432,15 @@ func (r *splunkReceiver) handleReq(resp http.ResponseWriter, req *http.Request)
 		return
 	}
 
+	var channelID string
+	var extracted bool
```

Comment:
Does extracted need to be declared as a var?

Reply:
removed